configure

```diff
@@ -1518,9 +1601,12 @@ fi
 if [ -n "$MONGODB_IPATH" ]; then
     XIPATHS="$XIPATHS -I$MONGODB_IPATH -I$BSON_IPATH"
 fi
-if [ -n "$FREERDP2_IPATH" ]; then
+if [ -n "$FREERDP3_IPATH" ]; then
```

Comment:
this one is wrong, you moved the check to RDP3, but then copy RDP2_IPATH. as the real RDP3 one is below this has to be $FREERDP2_IPATH :)

Reply:
this bug is still present

Reply:
Working on this today and will update when finished